### PR TITLE
Austin.mcdaniel/add securables endpoints

### PR DESCRIPTION
**Pull request by austinmcdaniel**
Adds support for the new EMS Securables endpoints for checking EMS permissions against specific securable items.

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 * Expose the "next maintenance window" in the EMS system info.
 * Include support for the Admin User GetUsers route.
 # Release 2.4.1
-* Adds support for the new analytic set endpoints.
\ No newline at end of file
+* Adds support for the new analytic set endpoints.
+# Release 2.5.0
```
Comment (c-owens) [inline ReleaseNotes.md:6]: I failed to mention this in a previous PR but this file should only contain the newest release (so delete the previous stuff). It gets attached as the text for the github release.

File: src/Client/V2/Access/EmsSecurablesAccess.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        /// <param name="securableId">
+        /// The identifier of a specific EMS securable item.
+        /// </param>
+        /// <param name="accessRight">
+        /// The securable type-specific access right to check access against.
+        /// </param>
+        /// <param name="context">
+        /// The optional call context to include.
+        /// </param>
+        public virtual EmsSecurableEffectiveAccess GetAccessForSecurable(string securableId, string accessRight, CallContext context = null )
```
Comment (c-owens) [inline src/Client/V2/Access/EmsSecurablesAccess.cs:63]: Spacing on the first argument here

**Review by c-owens — COMMENTED** [review]
I'm guessing something needs to be setup on d1mo to fix the test failures, but let me know if that's not the case or if there's something you want me to check out

File: ReleaseNotes.md (hunk 1 above)
Comment (austinmcdaniel) [reply ReleaseNotes.md:1]: ah I was wondering about that last time, I figured we just didn't have this before. Should be fixed now.

File: src/Client/V2/Access/EmsSecurablesAccess.cs (hunk 2 above)
Comment (austinmcdaniel) [reply src/Client/V2/Access/EmsSecurablesAccess.cs]: fixed.

**Comment by austinmcdaniel** [comment]
I fixed up the test in a previous change -- I just adjusted which securable and access rights are used for the test. I think this is good to go now once this latest build succeeds.

**Pull request merged**